pkg/trace/api/api_oom_test.go

```diff
@@ -36,7 +36,7 @@ func TestOOMKill(t *testing.T) {
 	conf := config.New()
 	conf.Endpoints[0].APIKey = "apikey_2"
 	conf.WatchdogInterval = time.Millisecond
-	conf.MaxMemory = 0.5 * 1000 * 1000 // 0.5M
+	conf.MaxMemory = 0.4 * 1000 * 1000 // 0.4M
```

Comment:
why not just make this really tiny, like a couple KB, to make sure it gets killed as expecte?

Reply:
I'm not quite sure? My assumption was it was to be at least a "little" sure that the oom kill functionality doesn't activate too quickly?

Reply:
Sorry I didn't get back to you here! I forgot about this PR, and then when I noticed the flake, I went ahead and lowered the memory to make sure it triggers: https://github.com/DataDog/datadog-agent/commit/269c9dc9ff5fc0a77f2bdb5056691ec8d5adb464
You make a fair point that we want to make sure that the OOM kill functionality is working well. LMK if you want to close this PR or try to merge it anyway.